### DISCUSSION TITLE
Customize 15m Salt Checkin value

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#4054] [Unsupported Versions] [answered]
I shutdown one of my sensors to see how quickly the grid page would update from "OK" to "Fault", and it appears to be tied to the 15m salt checkin. Is this correct? Is there a way to get it to update more frequently w/o modifying the 15m default check in value?

**Comment by TOoSmOotH** [comment]
The sostatus runs once every minute. When a sensor boots nothing will start until it checks in with the manager. If you want to watch OK to Fault then go to a device and stop one of the docker containers that is not sensoroni and it should flip to a fault state. Once the highstate runs again it should change back to green.

**Comment by darth-drew** [comment]
Thanks for the info. I tested by running so-suricata-stop and start, and OK -> FAULT does happen within a minute or two, which is helpful. Is the "status" chip designed to alarm in the case a sensor takes a power hit? Would you recommend I create some kind of automated test.ping dashboard to detect for sensors which have been knocked offline? I'd hate to change the 15m highstate value, because I believe this grid is going to get pretty big.

**Reply by jertel** [reply]
It's not tied to the salt highstate interval. There's a separate setting in SOC, named `offlineThresholdMs`, that is also set to 15 minutes. Currently it's not pillarized which means if you changed it you would lose that change on the next SO upgrade.   15 minutes seemed like a reasonable amount of time but we could consider shortening it to 10 minutes, maybe even 5 minutes. But reducing to 1 or 2 minutes could cause false-positives should a packet get dropped, etc.

**Comment by darth-drew** [comment, marked as answer]
Thank you all for the info!
